src/ripple/app/ledger/impl/LedgerMaster.cpp

```diff
@@ -959,144 +969,22 @@ class LedgerMasterImp
         return *ret;
     }
 
-    // Try to publish ledgers, acquire missing ledgers
-    void doAdvance ()
+    bool shouldFetchPack (std::uint32_t seq)
```

Comment:
`shouldFetchPack` should probably be `const` and not change `fetch_seq_` Can we chage `fetch_seq_` in `getFetchPack` instead? If not then consider changing the name of this function to sound less like a query only function.